client/components/themes-list/index.jsx

```diff
@@ -48,9 +52,19 @@ export const ThemesList = ( props ) => {
 			{ props.themes.map( ( theme, index ) => (
 				<ThemeBlock key={ 'theme-block' + index } theme={ theme } index={ index } { ...props } />
 			) ) }
+			{ /* The Pattern Assembler CTA will display on the fourth row and the behavior is controlled by CSS */ }
+			{ isEnabled( 'pattern-assembler/logged-out-showcase' ) &&
+				props.themes.length > 0 &&
+				! isLoggedIn && (
+					<PatternAssemblerCta
+						onButtonClick={ () =>
+							window.location.assign(
+								`/start/with-theme?ref=calypshowcase&theme=${ BLANK_CANVAS_DESIGN.slug }`
```

Comment:
This is the theme parameter you are talking about p1675307895826869/1675301136.567349-slack-CRWCHQGUB, ya?

Reply:
Right